airflow/providers/amazon/aws/triggers/emr.py

```diff
@@ -242,7 +242,7 @@ def __init__(
         )
 
     def hook(self) -> AwsGenericHook:
-        return EmrContainerHook(self.aws_conn_id)
+        return EmrContainerHook(aws_conn_id=self.aws_conn_id)
```

Comment:
@vandonr-amz Not related to this PR, but why we decide to use `hook` as method instead of property here?

Reply:
Just wondering. Maybe some serialization issues or something like that

Reply:
I don't remember exactly, maybe it is because it actually builds the hook every time, so it shouldn't be used "like a property".

Reply:
For this case we usually decorate it as `@cahed_property`. In general it uses in a lot of places over the different providers, especially when you need to move something heavy from constructor and keep compatible attribute.

Anyway it was just a question, no offence

Reply:
yes, sure we can, but it shouldn't be necessary as we use it only in once place.